.env.defaults

```diff
@@ -36,8 +36,14 @@ FL_PARTNERS_CONSUMER_SECRET=secret
 FL_PARTNERS_URL=https://lticourses-api.sandbox.futurelearn.com # The sandbox is currently disabled
 FL_PARTNERS_IGNORED_COURSE_UUIDS=
 
+FLAG_NCCE2=on
+NCCE2_START_DATE='2023-04-02'
+
 MYLEARNING_DASHBOARD_URL=https://mylearning.stem.org.uk/my/
 
+FLAG_NCCE2_TEST=off
+FLAG_NCCE2_LIVE=off
```

Comment:
Just curious if these were intended to be left in, as they don't seem to be used as part of these changes?